webui.js

```diff
@@ -70,6 +82,47 @@ passport.use('local-signin', new LocalStrategy(
   }
 ));
 
+
+var OPTS = {
+  server: {
+    url: mineos_config['ldap']['url'],
```

Comment:
This line throws an error when I try to run. There is no 'url' field under 'ldap' in mineos.conf